sass/_themes.scss

```diff
@@ -395,29 +398,31 @@ $themes: (
         LastdigitsMinBarBg: $COLOR_RED,
         LastdigitsBarText: $COLOR_LIGHT_GRAY,
         LastdigitsBarFooter: $COLOR_BLACK,
-        ChartHistoryBg: $COLOR_LIGHT_GRAY_1,
-        ChartHistoryColor: $COLOR_LIGHT_BLACK,
+        ChartHistoryBg: $COLOR_LIGHT_GRAY_3,
+        ChartHistoryBgContainer: $COLOR_WHITE,
+        ChartHistoryColor: $color-black-1,
         ChartHistoryBorder: $COLOR_LIGHT_GRAY,
         ChartHistoryPickerBg: $COLOR_WHITE,
-        ChartHistoryPickerBorder: $COLOR_LIGHT_GRAY,
+        ChartHistoryPickerBorder: $color-grey-3,
         ChartHistoryPickerColor: $COLOR_LIGHT_BLACK,
         ChartHistoryPickerIcon: $COLOR_LIGHT_BLACK,
-        ChartHistoryPickerActiveBorder: $COLOR_DARK_BLUE_5,
+        ChartHistoryPickerActiveBorder: $COLOR_GREEN,
         ChartHistoryPickerActiveColor: $COLOR_LIGHT_BLACK,
         ChartHistoryPickerActiveIcon: $COLOR_DARK_BLUE_5,
         ChartHistoryPickerTopColor: $COLOR_BRANDBLUE_40,
         ChartHistoryPickerHeaderBg: $COLOR_WHITE,
-        ChartHistoryPickerHeaderBorder: $COLOR_LIGHT_GRAY,
+        ChartHistoryPickerHeaderBorder: $COLOR_LIGHT_GRAY_3,
         ChartHistoryPickerHeaderColor: $COLOR_BLACK,
-        ChartHistoryPickerHeaderIcon: $COLOR_BRANDBLUE_40,
+        ChartHistoryPickerHeaderIcon: $color-black-1,
         ChartHistoryPickerFooterBg: $COLOR_WHITE,
-        ChartHistoryPickerFooterBorder: $COLOR_LIGHT_GRAY,
-        ChartHistoryPickerFooterColor: $COLOR_DARK_BLUE_5,
-        ChartHistoryPickerBodyColor: $COLOR_DARK_BLUE_5,
-        ChartHistoryPickerBodyDisableColor: $COLOR_DARK_GRAY_3,
-        ChartHistoryPickerBodyActiveBg: $COLOR_DARK_BLUE_5,
-        ChartHistoryPickerBodyActiveColor: $COLOR_WHITE,
+        ChartHistoryPickerFooterBorder: $COLOR_LIGHT_GRAY_3,
+        ChartHistoryPickerFooterColor: $color-black-1,
+        ChartHistoryPickerBodyColor: $color-black-1,
```

Comment:
Can we transform all these colors to `CAPITAL LETTERS`? Seems this is what mostly used for at least `ChartHistoryPicker` colors.

Reply:
Fixed

Reply:
Seems like `COLOR PALETTE` is not consistent as some of them are in capital and the reset are not, please help to make it consistent!

Reply:
@matin-binary do you mean by consistent to do in upper case all color palette?

Reply:
color palette section seems to have all in small letters so keep it consistent. no need to put them in capital letter.

Reply:
Fixed